src/content/en/tools/workbox/index.md

```diff
@@ -0,0 +1,177 @@
+
+## Generating a Service Worker
+
+The [Workbox command-line tool](https://github.com/GoogleChrome/workbox/tree/master/packages/workbox-cli)
+(`workbox-cli`) is a standalone CLI to generate a service worker and a file
+manifest, that uses the `workbox-sw` module to end up with a fully-functioning
+service worker.
+
+Don't want to use yet another CLI? Workbox is also compatible with many
+different build tools. The
+[`workbox-build`](https://github.com/GoogleChrome/workbox/tree/master/packages/workbox-build)
+module is a portable Node module that can generate a file manifest or service
+worker to work with `workbox-sw`. The `workbox-build` module can be used with
+Gulp, NPM scripts, or many other Node-based build tools.
+
+Workbox also includes a
+[Webpack Plugin](https://github.com/GoogleChrome/workbox/tree/master/packages/workbox-webpack-plugin)
+(`workbox-webpack-plugin`), which is exactly what it sounds like - a plugin for
+Webpack for generating a service worker.
+
+### Features
+
+| Feature | Summary |
+|---------|---------|
+| Precache Your App Shell | Your web app's shell—its core HTML, JavaScript, and
+CSS—can be precached when a user visits your page. |
+| Build-time Integration | Drop it into your existing build process. |
+| Stay Fresh | Changes in your build update the service worker script. Users get
+updates, but you don't have to manually version your content or caches. |
+| No Network, No Problem | Your static resources are served
+[cache-first](/web/fundamentals/instant-and-offline/offline-cookbook/#cache-falling-back-to-network)
+, quickly, whether or not there's a network available. |
+
+
+## Offline Google Analytics
+
+[Workbox Google Analytics](https://github.com/GoogleChrome/workbox/tree/master/packages/workbox-google-analytics)
+temporarily holds and retries analytics requests to avoid losing them to network
+disconnections. This tool easily installs to your build system using npm and is
+easily imported into your service worker script. Configure it using a
+parameterized function call.
+
+### Features
+
+| Feature | Summary |
+|---------|---------|
+| Offline Google Analytics | Creates fetch handlers that ensure the Google
+Analytics JavaScript is available offline. |
+| Temporarily Caches Data | Holds analytics requests that are made when the
+device is offline and retries them the next time the service worker starts up. |
+| Custom Replay Values | Key/value pairs to be added to replayed Google
+Analytics requests. For example, you might set a custom dimension to indicate
+that a request was replayed. |
+| Modified Hit Parameters | Lets you programmatically modify a hit's parameters
+to, for example, track the elapsed time between when a hit is attempted and when
+ it is replayed. |
+
+## Background Sync
+
+[Workbox Background Sync](https://github.com/GoogleChrome/workbox/tree/master/packages/workbox-background-sync)
+will queue failed requests and use the
```

Comment:
Use present tense verbs here to be consistent with the rest of your descriptions.